joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForPackageConstructorCreator.scala

```diff
@@ -1,12 +1,13 @@
 package io.joern.gosrc2cpg.astcreation
 
+import flatgraph.DiffGraphBuilder
 import io.joern.gosrc2cpg.parser.ParserAst.Unknown
 import io.joern.gosrc2cpg.parser.ParserNodeInfo
 import io.joern.x2cpg.astgen.AstGenNodeBuilder
 import io.joern.x2cpg.{Ast, AstCreatorBase, ValidationMode, Defines as XDefines}
 import io.shiftleft.codepropertygraph.generated.NodeTypes
+import flatgraph.DiffGraphBuilder
 import org.apache.commons.lang3.StringUtils
-import overflowdb.BatchedUpdate.DiffGraphBuilder
```

Comment:
Can we improve compat here?

I.e. introduce a type-alias somewhere now (e.g. in the generated domain classes), use that, merge this change beforehand and independently of the big flatgraph change?

I'm trying to think about ways of making the diff more digestible.

Reply:
Hmm, sure we could do that, but I'm not sure that it would really help. 95% of the changes are essentially noise that we could remove by adapting odb-codegen and odb. And I know that your ideal is to have a drop-in replacement, but personally I'm not convinced that the additional work is really worth it. I kept rebasing this PR over 6 months or so and it's never been an issue. Maybe I'm just lazy, but IMO that's a good trait 🤷🏻 😸